exercises/practice/leap/.meta/tests.toml

```diff
@@ -0,0 +1,37 @@
+# This is an auto-generated file.
```

Comment:
This PR currently adds a `tests.toml` file for `leap`, but it looks like this PR is just supposed to implement `all-your-base`.

Reply:
was intended just to implement all-your-base. In the running of configlet sync -u, leap's toml was generated.
Then forgot to create another branch and implemented resistor-color